src/main/scala/wdl4s/values/WdlInteger.scala

```diff
@@ -5,7 +5,7 @@ import wdl4s.types.WdlIntegerType
 
 import scala.util.{Failure, Success, Try}
 
-case class WdlInteger(value: Integer) extends WdlPrimitive {
+case class WdlInteger(value: Int) extends WdlPrimitive {
```

Comment:
This is a spinoff of https://github.com/broadinstitute/cromwell/pull/2031

In this IntelliJ screenshot:
<img width="882" alt="toInt implicit underlined" src="https://cloud.githubusercontent.com/assets/791985/23348863/129fe78c-fc7c-11e6-81a9-4541225aacd4.png">

That implicit underlining the `.toInt` on line 178 shouldn't be there.